test/app/views/teachers/EditStudentModal.spec.coffee

```diff
@@ -0,0 +1,65 @@
+EditStudentModal = require 'views/teachers/EditStudentModal'
+User = require 'models/User'
+factories = require 'test/app/factories'
+
+describe 'ActivateLicensesModal', ->
+
+  user = null
+  modal = null
+  email = "test@example.com"
+  newPassword = "new password"
+
+  describe 'for a verified user', ->
+    beforeEach (done) ->
+      user = factories.makeUser({ email, emailVerified: true })
+      modal = new EditStudentModal({ user })
+      jasmine.demoModal(modal)
+      modal.render()
+      _.defer done
+
+    it 'has a button to send a password reset email', ->
+      if modal.$('.send-recovery-email-btn').length < 1
+        fail "Expected there to be a Send Recovery Email button"
+
+    it 'sends the verification email request', ->
+      modal.$('.send-recovery-email-btn').click()
+      request = jasmine.Ajax.requests.mostRecent()
+      expect(request.params).toEqual("email=#{encodeURIComponent(email)}")
+
+    it 'updates the button after the request is sent', ->
+      modal.$('.send-recovery-email-btn').click()
+      request = jasmine.Ajax.requests.mostRecent()
+      request.respondWith({ status: 200, responseText: "{}" })
+      expect(modal.$('.send-recovery-email-btn').text()).toEqual('Email sent')
+
+  describe 'for an unverified user', ->
+    beforeEach (done) ->
+      user = factories.makeUser({ email , emailVerified: false })
+      modal = new EditStudentModal({ user })
+      jasmine.demoModal(modal)
+      modal.render()
+      _.defer done
+
+    it "has a new password field", ->
+      modal.render()
```

Comment:
This is needed even with the `modal.render()` called in `beforeEach`? I wouldn't be surprised if it is...